src/routes/index.svelte

```diff
@@ -235,7 +235,7 @@
       <div class="bx--row">
         <div class="bx--col-lg-12">
           <h3>
-            Discover how our <a href="/grants" target="_blank"
+            Discover how our <a href="/grants"
```

Comment:
@melijimenez11 For some reason I no longer see the scrolling bug, so I removed the `target=_blank`.

Reply:
Sounds good